include/psa/crypto_sizes.h

```diff
@@ -992,9 +992,11 @@
  */
 #define PSA_CIPHER_ENCRYPT_OUTPUT_SIZE(key_type, alg, input_length)             \
     (alg == PSA_ALG_CBC_PKCS7 ?                                                 \
+     (((key_type) & PSA_KEY_TYPE_CATEGORY_MASK)                                 \
+                                           == PSA_KEY_TYPE_CATEGORY_SYMMETRIC ? \
```

Comment:
While this is correct, `PSA_BLOCK_CIPHER_BLOCK_LENGTH(key_type)` would make the correctness more obvious.